gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -48,6 +48,12 @@
         public final List<String> gcJvmOpts() {
             return ImmutableList.of("-XX:+UseParNewGC",
                     "-XX:+UseConcMarkSweepGC",
+                    /*
+                     * When setting UseConcMarkSweepGC the default value of NewRatio (2) is completely ignored.
+                     *
+                     * https://bugs.openjdk.java.net/browse/JDK-8153578
+                     */
+                    "-XX:NewRatio=2",
```

Comment:
Wanna add a setter the same way as `initiatingOccupancyFraction` so it's possible to modify this